cmd/maya-exporter/app/collector/metrics.go

```diff
@@ -127,112 +137,211 @@ type MetricsDiff struct {
 // MetricsInitializer returns the Metrics instance used for registration
 // of exporter while instantiating JivaStatsExporter and
 // CstorStatsExporter.
-func MetricsInitializer() *Metrics {
+func MetricsInitializer(casType string) *Metrics {
 	return &Metrics{
```

Comment:
Re-order the metrics into:
1) Uptime
2) Capacity-relate metrics
3) Performance-related metrics grouped by reads and writes.